CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+### Breaking changes
+
+- `target` block in `prometheus.exporter.blackbox` requires mandatory `name`
+  attribute instead of label block previously. (@hainenber)
```

Comment:
```suggestion
- The `target` block in `prometheus.exporter.blackbox` requires a mandatory `name`
  argument instead of a block label. (@hainenber)
```